src/time-picker/TimePicker.js

```diff
@@ -155,6 +156,8 @@ export const filterTime = ({ originalInput, minTime, maxTime }) => time => {
   return lteTime(minTime, time) && lteTime(time, maxTime) && startsWith(time, originalInput);
 };
 
+export const sortTimeList = timeList => sortBy(timeList, time => ( time.hours === 0 ? 24 : time.hours));
```

Comment:
aren't you breaking the order of H24 options this way?
Also, why are you exporting it? If you plan to improve tests you can:
- do it directly in this PR
- open an issue